test/babel/snapshots/index.js.md

```diff
@@ -111,9 +111,18 @@ Generated by [AVA](https://ava.li).
     };␊
     const small = true;␊
     const styles1 = {␊
-      root: ["dss10_h28rbs-i0tgik", "dss10_1buiceq-7qvd50", "dss10_f69c5w-oyp9nw", "dss10_u84hpd-dby8y8", "dss10_1avj1fl-7qvd50", "dss10_153t8jg-7qvd50", "dss10_1eznbv4-6laobc", "dss10_1jgjtkn-1k19bls"],␊
       foo: ["dss10_j9ctud-1t7uh5u"],␊
       ..._styleSheet.StyleSheet.create({␊
+        root: {␊
+          color: 'red',␊
+          marginTop: marginTop,␊
+          marginBottom: marginTop + 'px',␊
+          paddingTop: marginTop / 2,␊
+          fontSize: small ? marginTop : 4,␊
+          val: _constants.TEST,␊
+          backgroundColor,␊
+          filter: 'blur(10px)'␊
+        },␊
```

Comment:
@chrisands we should figure out why this new version of linaria is not compiling these anymore, or maybe at which version after 1.3.1 it started to break

Reply:
I would suggest that https://github.com/callstack/linaria/commit/d57e859bff7eaf85cdf7c19d9b421e81aace44ce commit introduced breaking changes.

Reply:
I will have to verify, maybe I find some time over the weekend. FYI we only use the `evaluate` module from linaria